debug.go

```diff
@@ -32,12 +33,9 @@ var (
 	allocatorDev      = false
 )
 
-var READMEMSTATS = true
-
+// TABCOUNT is a global flag used when debugging
```

Comment:
Don't know what it does

Reply:
readmemstats is an old variable used to dump memory statistics. Not used any more